scripts/import/uniprot/uniprot_importer.py

```diff
@@ -139,29 +160,31 @@ def insert_uniprot_data(db_host, db_port, db_name, db_user, db_password, total_i
     db.commit()
     db.close()
     print("Uniprot data successfully inserted into G2P database.")
-    print(f'Total Uniprot entries fetched: {len(total_items)}')
-    print(f'Total Uniprot entries inserted: {insert_count}')
+    print(f"Previous total number of uniprot entries: {previous_number_rows[0]}")
+    print(f"Total Uniprot entries fetched: {len(total_items)}")
+    print(f"Total Uniprot entries inserted: {insert_count}")
     print("Note: Only Uniprot data entries with existing Gene information in the database will be inserted.")
 
+
 def main():
-    # Fetch Database configuration from User
     parser = argparse.ArgumentParser(description="")
-    parser.add_argument("--g2p_host", required=True, help="G2P database host")
-    parser.add_argument("--g2p_port", required=True, help="G2P database host port")
-    parser.add_argument("--g2p_database", required=True, help="G2P database name")
-    parser.add_argument("--g2p_user", required=True, help="G2P database Username")
-    parser.add_argument("--g2p_password", default='', help="G2P database Password (default: '')")
+    parser.add_argument("--config", required=True, help="Config file with details to the G2P database")
 
     args = parser.parse_args()
+    config_file = args.config
+
+    # Load the config file
+    config = configparser.ConfigParser()
```

Comment:
And also if we could make this a different function

Reply:
Are you suggesting to move the config loading into a separate function?
The function would have ~10 lines and be called only once.

Reply:
Yeah but it is easier for modularity and for readability and also in the future "testing"

Reply:
Same as https://github.com/EBI-G2P/gene2phenotype_scripts/pull/12#discussion_r2068316503